lib/widgets/core/screens/screen_movie_details.dart

```diff
@@ -0,0 +1,154 @@
+import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+import '/widgets/custom_widgets.dart';
+
+class ScreenMovieDetails extends StatefulWidget {
+  const ScreenMovieDetails({super.key});
+
+  @override
+  State<ScreenMovieDetails> createState() => _ScreenMovieDetailsState();
+}
+
+class _ScreenMovieDetailsState extends State<ScreenMovieDetails> {
+  int _counter = UiConstants.kCounterValueZero;
+  final ButtonStyle buttonStyle = ElevatedButton.styleFrom(
+    backgroundColor: Colors.white,
+  );
+  final movie = Movie(
+    title: UiText.movieFullTitle,
+    releaseDate: UiText.movieReleaseDate,
+    rating: UiText.movieRating,
+    largePhotoURL: UiText.networkImageWidePhotoLink,
+    smallPhotoURL: UiText.networkImageSmallPhotoLink,
+  );
+
+  @override
+  void initState() {
+    super.initState();
+    WidgetsBinding.instance.addPostFrameCallback(
+      (_) async {
+        sharedPreferenceInstance = await SharedPreferences.getInstance();
+        _changeCounter(
+          CounterAction.load,
+        );
+      },
+    );
+  }
+
+  late final SharedPreferences sharedPreferenceInstance;
+
+  final List<String> genres = [
+    UiText.movieGenreAnimation,
+    UiText.movieGenreFamily,
+    UiText.movieGenreAdventure,
+    UiText.movieGenreFantasy,
+    UiText.movieGenreComedy
+  ];
```

Comment:
The genres list should be a property of the movie, as they're related to this particular movie!